src/components/EventDetailsComponents/components/Extras.tsx

```diff
@@ -430,6 +440,11 @@ const styles = StyleSheet.create({
   },
   extrasGalleryContainer: {
     flex: 1,
+    alignItems: 'center',
+  },
+  extrasGalleryContainerInner: {
```

Comment:
you need to create something like this example  extrasGalleryLastItemContainer: {
    marginRight: scaleSize(147),
  },

and add it to the last item in the list.  if the length of the list is more than one